engine/src/main/java/org/terasology/recording/EventCatcher.java

```diff
@@ -21,7 +21,8 @@
 import java.util.List;
 
 /**
- * This class is responsible for catching the events during a Record and send the desired ones to the RecordedEventStore.
+ * Gets the events sent in the EventSystemImpl and adds their copies to the RecordedEventStore if they are of a type
+ * selected to be recorded.
```

Comment:
How would you find out whether something is selected to be recorded? Would be a great spot for a javadoc link to an associated class :-)